keyboards/cornia/keymaps/default/keymap.c

```diff
@@ -0,0 +1,83 @@
+/* Copyright 2024 Vaarai
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+/* Custom Keycodes (CK_xxx) */
+#define CK_LPAR LSFT(KC_9)
+#define CK_RPAR LSFT(KC_0)
+#define CK_LCBR LSFT(KC_LBRC)
+#define CK_RCBR LSFT(KC_RBRC)
+#define CK_QMRK LSFT(KC_SLSH)
+#define CK_UNSC LSFT(KC_MINS)
+
+/* Layers definitions */
+enum layers {
+  _ALPHA,
+  _NAV,
+  _NUM,
+  _ADJUST,
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_ALPHA] = LAYOUT_split_3x6_3(
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+       KC_ESC,    KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,                         KC_Y,    KC_U,    KC_I,    KC_O,    KC_P, KC_BSPC,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+       KC_TAB,    KC_A,    KC_S,    KC_D,    KC_F,    KC_G,                         KC_H,    KC_J,    KC_K,    KC_L, KC_SCLN, KC_QUOT,
+  //|--------+--------+--------+--------+--------+--------|                    |--------+--------+--------+--------+--------+--------|
+      KC_LCTL,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,                         KC_N,    KC_M, KC_COMM,  KC_DOT, KC_SLSH, KC_RCTL,
+  //|--------+--------+--------+--------+--------+--------+--------|  |--------+--------+--------+--------+--------+--------+--------|
+                                          KC_LSFT,MO(_NAV),  KC_SPC,     KC_ENT,MO(_NUM), KC_RSFT
```

Comment:
Rather than this, you might want to enable the tri layer feature, and use those keycodes. 
https://docs.qmk.fm/features/tri_layer